auth/src/main/java/software/amazon/awssdk/auth/credentials/AwsCredentialsProviderChain.java

```diff
@@ -106,8 +106,10 @@ public AwsCredentials resolveCredentials() {
             }
         }
 
-        throw new SdkClientException("Unable to load credentials from any of the providers in the chain " + this
-                + ": " + exceptionMessages);
+        throw SdkClientException.builder()
+                                .message("Unable to load credentials from any of the providers in the chain " +
+                                         this + " : " + exceptionMessages)
+                                .build();
```

Comment:
Can we have `create` methods that take 1, 2 parameters, since specifying message and cause is such a common thing to do?

Reply:
Ping

Reply:
Create method added on SdkException and SdkClientException. Can add to the others but the rest typically have additional fields as well and not just message / cause.